cmd/internal/shared/config.go

```diff
@@ -107,7 +107,9 @@ var DefaultRightsConfig = config.Rights{
 }
 
 // DefaultKeyVaultConfig is the default config for key vaults.
-var DefaultKeyVaultConfig = config.KeyVault{}
+var DefaultKeyVaultConfig = config.KeyVault{
+	Provider: "static",
```

Comment:
This is the only one we can do, because before this version there's only `static`